docs/datagrid/datagrid-v2-migration.md

```diff
@@ -1,12 +1,11 @@
-Upgrade DataGrid to v3
-===
+# Upgrade DataGrid to v3
 The core of DataGrid has been changed in version 3.0. We have increased the performance and flexibility while adding accessibility support. This means that API options have been changed and the definition of styles or data source are different. We outline all of the changes below.
```

Comment:
"This means that API options have been changed and the definition of styles or data source are different".

Suggested rewrite: This means that API options and the definition of styles or data source differ from the previous version.